pkg/bucketeer/sdk.go

```diff
@@ -204,100 +240,272 @@ func newSegmentUserCacheProcessor(
 }
 
 func (s *sdk) BoolVariation(ctx context.Context, user *user.User, featureID string, defaultValue bool) bool {
+	return s.BoolVariationDetail(ctx, user, featureID, defaultValue).VariationValue
```

Comment:
Other interfaces have also been changed to utilize `VariationDetail()`.
https://github.com/bucketeer-io/go-server-sdk/pull/136#discussion_r1686542852